src/main/java/org/opentripplanner/routing/alertpatch/AlertPatch.java

```diff
@@ -82,6 +79,23 @@ public boolean displayDuring(State state) {
         return false;
     }
 
+    private boolean serviceAffected() {
+        GtfsRealtime.Alert.Effect effect = getAlert().getEffect();
+        return effect.equals(GtfsRealtime.Alert.Effect.NO_SERVICE) || effect.equals(GtfsRealtime.Alert.Effect.DETOUR);
```

Comment:
I don't know this data model this well, so perhaps it's impossible to have a `null` here. Still, out of abundance of caution maybe: `GtfsRealtime.Alert.Effect.NO_SERVICE.equals(effect)` instead?